src/Analysis.cpp

```diff
@@ -87,6 +88,8 @@ struct Analysis
     {
         int count = 0;
         int64_t us = 0;
+        DetailIndex file;
+        std::unordered_map<int, int> templateCount;
```

Comment:
I don't like this approach, but I need guidance on where these things should go.